azure-pipelines.yml

```diff
@@ -0,0 +1,53 @@
+# Based on a Starter pipeline
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+# Optional to only trigger on Master branch changes
+# trigger:
+# - master
+
+# Define the continaers we want access to
+resources:
+  containers:
+    -   container: 'qcengineci' # Arbitrary name for reference in pipelines
+        image: qcarchive.azurecr.io/qcengineci:latest # pointer to the Image in our Azure CR
+        endpoint: "QCArchive Azure Container Registry" # Name of the Service Connection the pipeline is configured with
```

Comment:
Can you describe a bit about how the authentication is handled? Is this a private Azure pw?